### ISSUE TITLE
Data-scroll "in" by default, working in wrong direction.

### ISSUE THREAD

**Issue by iamrasmus** [scroll-out/scroll-out#49]
I'm having a problem with this script. If the page is loaded, then ALL data-scroll elements are classed as "in" even though these are not seen on the screen yet. Scrolling from top to the bottom won't give any transitions. But when i finally scroll past to the data-scroll element, it changes to the "out" (as it is supposed to). Script seems to work in the opposite direction of the page, because when scrolling from bottom to the top, then data-scroll element changes from "out" to "in" and transition works. 

Any clues?

**Comment by notoriousb1t** [comment]
Usually these types of problems are fixed by setting scrollingElement in the ScrollOut() options. The default is the documentElement. I would start by figuring out what element is scrolled and passing it into scrollingElement.

The element that is scrolled should have a different a clientHeight that is different than the scrollHeight property.

**Comment by jonwilcox** [comment]
@iamrasmus Did you ever solve this? I am having the same issue. Everything is "in" when the page loads until I scroll down past it. Then it changes to "out". Scrolling back to the top changes them to "in" again. 

I haven't changed anything that should affect the default scrollingElement. That being said, I checked the clientHeight and scrollHeight of document.documentElement and they are the same. scrollHeight is great than clientHeight for document.body, but when I set the document.body as the scrollingElement in options, I have different problem. [data-scroll] attributes seem to get set properly when the page loads. Items that are in view get set to "in" and vice versa. But they don't change when the page is scrolled.

**Comment by jonwilcox** [comment]
I think I may have solved this, at least for my instance. I finally realized my page was missing a doctype. Once I added `<!DOCTYPE html>` scroll-out starting calculating "in" and "out" properly.

**Comment by notoriousb1t** [comment]
I'm surprised that this was affected by the html version. Did you see this behavior across browsers and if so, which ones?

**Comment by jonwilcox** [comment]
I did see it in both Firefox and Chrome for Mac. I was working on a new template for my site when I noticed it. I had used ScrollOut before on my site without issue, so I found this very odd. I took the exact same html and JS from my new template and put them into a CodePen. ScrollOut worked there. When I inspected the source code of both pages in my browser, that's when I realized the template on my site was missing the doctype, but it was present on CodePen. As soon as I added the doctype, everything worked as expected. I assume the missing doctype put the page in Quirks mode and at that point all bets are off.
